test/cli-test/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 	for _, arg := range flag.Args() {
 		cmd := exec.Command("bash", arg)
 		if os.Getenv("TMPDIR") != "" {
+			cmd.Env = append(cmd.Env, os.Environ()...)
```

Comment:
on my machine (darwin/arm64), I have a non-standard PATH env, so I needed to add this in order to tests pass. happy to revert